shared-libs/message-utils/src/index.js

```diff
@@ -96,21 +96,52 @@
   return phone;
 };
 
+const normalizeRecipient= function(recipient) {
```

Comment:
I see that there are at least 3 places where we test for string and check for validity and it's confusing and messy. 

I suggest we convert this function to properly normalize the recipient in the sense that: 

- converts the recipients to strings and also does trimming
- filters empty strings
- returns an array even when there is only one recipient

That saves us from all the additional checks later.